src/net/atlas/download.rs

```diff
@@ -98,6 +98,9 @@ impl AttachmentsDownloader {
                         .resolve_attachment(&attachment.hash())
                 }
 
+                // Every once in a while, we delete uninstanciated attachments
+                network.atlasdb.evict_expired_uninstantiated_attachments()?;
```

Comment:
Do you know how often this will run?  Is it once per state-machine pass (e.g. once every 5 calls to `PeerNetwork::run()`), or is it unbound?

Reply:
> once per state-machine pass

Yes.